src/components/molecules/card/card.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<div class="card-body">
+  <div class="box-image">
+    <img src="../../../img/card-1.jpg" class="card-img">
```

Comment:
```suggestion
  <div class="img-box">
    <img src="../../../img/card-1.jpg" class="img-card">
```
scssとhtmlでクラス名が違うので、スタイルが反映されていない感じです！